app/js/lib/foxbox/webpush.js

```diff
@@ -80,10 +80,10 @@ export default class WebPush {
       })
       .catch((error) => {
         if (Notification.permission === 'denied') {
-          throw 'Permission request was denied.';
+          throw new Error('Permission request was denied.');
         } else {
```

Comment:
__nit:__ Ha! I expected ESLint to complain about ```else``` if we clearly have only throw in the ```if``` section :)